package-lock.json

```
@@ -1,7 +1,7 @@
 {
   "name": "@openinf/gh-file-importer",
   "version": "2.0.1",
-  "lockfileVersion": 2,
+  "lockfileVersion": 3,
```

```suggestion
  "lockfileVersion": 3,
```

This change (and ones associated w/ it) causes the Snyk check to fail due to a de-sync error&hellip;